contracts/KIP/token/KIP7/KIP7.sol

```diff
@@ -235,7 +235,7 @@ contract KIP7 is Context, KIP13, IKIP7, IKIP7Metadata {
      * - the caller must have a balance of at least `amount`.
      * - if `recipient` is a smart contract, it must implement {IKIP7Receiver}
      */
-    function safeTransfer(address recipient, uint256 amount) public {
+    function safeTransfer(address recipient, uint256 amount) public override {
```

Comment:
Don't we need add virtual as well?
https://github.com/klaytn/klaytn-contracts/blob/master/contracts/KIP/token/KIP17/KIP17.sol#L170

Reply:
yes we need to add `virtual` keyword as well.

Reply:
I added `virtual` keyword too